roles/timer/templates/timer

```diff
@@ -0,0 +1,20 @@
+# {{ ansible_managed }}
+
+[Unit]
+Description={{ timer_name }} Service
+
+[Timer]
+Unit={{ timer_name }}.service
+{% if timer_calendar is defined %}
+OnCalendar={{ timer_calendar }}
+{% else %}
+OnBootSec={{ timer_startup_delay | default("60") }}
```

Comment:
please put defaults into `roles/timer/default/main.yml`

Reply:
There are the defaults.

Reply:
done